docs/tutorials/amd-igpu-780m.md

```diff
@@ -0,0 +1,116 @@
+
+The iGPU 780M of AMD Ryzen CPU could run on ROCm at Linux with extral settings. 
+
+Available CPU: Ryzen 7000s/8000s CPU with iGPU 780M
+
+## Test Platform
+
+| **Platform ** | **miniPC**                                                      |
+| ------------- | --------------------------------------------------------------- |
+| **HW**        | AMD Ryzen R8845HS + Radeon780M(iGPU, set 16GB VRAM of 64GB DDR) |
+| **OS**        | Ubuntu22.04                                                     |
+| **SW**        | ROCm6.0+PyTorch                                                 |
+
+
+## Prerequisites
+0. Set UMA for iGPU in BIOS.
```

Comment:
I believe this is the step where the user needs to dedicate a certain amount of system memory to the iGPU, correct?   We should note that leaving 1G wont work as our current code will skip the iGPU as a result, so at least > 1G has to be assigned, but really we should recommend more I think.

Reply:
OK. I will modify this part according your comments.